src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  messageNothing: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    messageNothing: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handler);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handler);
+  }
+
+  handler = (event: KeyboardEvent) => {
+    this.setState({
+      messageNothing: `The last pressed key is [${event.key}]`,
```

Comment:
you set `message` to property with key `messageNothing` - a bit inconsistently)

do it in this way: create `pressedKey` property in the state (as required) and save there **only** `event.key` (empty string by default)
then, using [conditional rendering/ternary operator](https://react.dev/learn/conditional-rendering) render the message you need